src/command_ui.cc

```diff
@@ -283,6 +283,56 @@ torrent::Object apply_equal(rpc::target_type target, const torrent::Object::list
   return result.is_value() ? result.as_value() == 0 : (int64_t)false;
 }
 
+torrent::Object
+apply_compare(rpc::target_type target, const torrent::Object::list_type& args) {
+  if (!rpc::is_target_pair(target))
+    throw torrent::input_error("Can only compare a target pair.");
+
+  if (args.size() < 2)
```

Comment:
There's no need to require this, make it work with zero compare fields.

Also, the order field should allow empty object or no args at all.